src/components/Tool/FixMailingAddresses/FixMailingAddresses.test.tsx

```diff
@@ -135,7 +135,7 @@ describe('FixMailingAddresses', () => {
     expect(
       getByText('You have 3 mailing addresses to confirm.'),
     ).toBeInTheDocument();
-    expect(getByText(/12\/10\/2022/i)).toBeInTheDocument();
+    await waitFor(() => expect(getByText(/12\/10\/2022/i)).toBeInTheDocument());
```

Comment:
Use  `findByText` as it's better since it's the same thing but less code.
```suggestion
    await findByText(/12\/10\/2022/i).toBeInTheDocument();
```